src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -139,21 +125,19 @@ public bool TryGetQuantityCodeId(string code, out int quantityCodeId)
             return _quantityCodeToId.TryGetValue(code, out quantityCodeId);
         }
 
-        public ValueTask EnsureInitialized() => _initializationOperation.EnsureInitialized();
-
-        private async Task Initialize()
+        protected override async Task Initialize()
         {
-            if (!_schemaInformation.Current.HasValue)
+            if (!SchemaInformation.Current.HasValue)
```

Comment:
Perhaps this should move to the base class since minimally, it should initialize the schema.

Reply:
Moved to common code in separate PR